WalletWasabi/Crypto/StringCipher.cs

```diff
@@ -1,114 +1,115 @@
-using System.IO;
 using System.Linq;
 using System.Security.Cryptography;
 using System.Text;
 using WalletWasabi.Crypto.Randomness;
 
 namespace WalletWasabi.Crypto
 {
-	// https://stackoverflow.com/a/10177020/2061103
```

Comment:
I would still mention https://tomrucki.com/posts/aes-encryption-in-csharp/ as it's useful to understand this code, otherwise the knowledge will be lost in a few months.